### ISSUE TITLE
call_user_func_array with by reference parameter

### ISSUE THREAD

**Issue by ponchofiesta** [kalessil/phpinspectionsea#715]
Actually by using call_user_func_array with a parameter by reference the inspection telling me I should use call_user_func instead:

```
\call_user_func_array($test->func, [&$param]);
```
![image](https://user-images.githubusercontent.com/12248938/34256884-b43a77cc-e657-11e7-89c3-556197428e2f.png)

But call_user_func doesn't support pass by reference. You have to use call_user_func_array with an array with references. So this suggestion can cause errors if you don't mind that. 

I assume this problem is more relevant when using objects as parameters but I didn't test it.

**Comment by funivan** [comment]
Confirmed. 
```php
function increment(&$var){
    $var++;
}

$a = 0;
call_user_func_array('increment', [&$a]); // <<-- works
call_user_func('increment', &$a); // <-- Parse error: syntax error, unexpected '&' 

```

**Comment by kalessil** [comment]
Thank you for reporting @ponchofiesta and checking @funivan @rentalhost, I'll fix it

**Comment by kalessil** [comment]
Fixed

**Comment by funivan** [comment]
@kalessil I see you are closing issues manually. Github can do this automatically

https://help.github.com/articles/closing-issues-using-keywords/

**Comment by ponchofiesta** [comment]
That was fast.
Thanks

**Comment by kalessil** [comment]
@funivan : I need to update labels and ensure commits gets linked to the ticket as well =)

@ponchofiesta : you are welcome, ETA release date for this change is somewhere in January =)
